test/e2e/checkpoint_test.go

```diff
@@ -933,18 +935,23 @@ var _ = Describe("Podman checkpoint", func() {
 	})
 
 	It("podman checkpoint and restore container with different port mappings", func() {
-		localRunString := getRunString([]string{"-p", "1234:6379", "--rm", redis})
+		rand.Seed(time.Now().UnixNano())
+		orginalPort := 37000 + rand.Intn(2000) + GinkgoParallelProcess()
```

Comment:
Any reason not to use `utils.GetRandomPort()`?  Both here and in line 968 instead of `+100`? (sorry for lateness; my earlier attempt to send this got rejected by github)

Reply:
Ah, thanks. I was not aware of it. I will use that function.

Reply:
Switched to the existing GetRandomPort() function.